### PR TITLE
Added drivers composite

### PR DESCRIPTION
**Pull request by piotr-s-nowak**

### CODE REVIEW

File: jobs2d.iml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kowallus) [inline jobs2d.iml]: unnecessary project file (REMOVE)

File: src/main/java/edu/kis/powp/jobs2d/drivers/CompositeDriver.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    @Override
+    public void operateTo(int x, int y) {
+        drivers.forEach(e->e.operateTo(x, y));
+    }
+
+    public void add(Job2dDriver driver){
+        drivers.add(driver);
+    }
+
+    public List getChildren(){
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/CompositeDriver.java:30]: Why without a template? 
It is better to return a list copy so it cannot be modified from outside.

**Review by kowallus — CHANGES_REQUESTED** [review]
Some code clean up required

**Comment by piotr-s-nowak** [comment]
changes commited

**Comment by piotr-s-nowak** [comment]
conflicts resolved

**Pull request merged**